auth/azure/client.go

```diff
@@ -51,6 +51,8 @@ type OptFunc func(*Client)
 // https://pkg.go.dev/github.com/Azure/azure-sdk-for-go/sdk/azidentity#NewDefaultAzureCredential
 // The default scope is to ARM endpoint in Azure Cloud. The scope is overridden
 // using OptFunc.
+//
+// Deprecated: Use auth.GetToken() with azure.Provider{} or NewTokenCredential().
```

Comment:
I propose we delete the client, the auth package is at v0.10.

Reply:
Fixed, thanks!